readme/git-pullrequest-workflow.md

```diff
@@ -10,26 +10,40 @@ Pick a name for your branch that tells us what it's for. Use this format:
 - `<issue number>`: This is the number of the task or issue in Github Projects you're working on.
 - `short-description`: This is a quick summary of what you're doing. Use hyphens instead of spaces.
 
+❗Please note - the `short-description` shouldn't contain numbers.
+
 Example: `feat/123-add-login-button`
 
 ## Commit Messages
 
 Your commit messages should be short, but they should tell us what changes you made. Use this format:
 
+`<type of work>: <summary>`
+
+Your commit message will be automatically converted to the following format:
+
 `<type of work>: <issue number> - <summary>`
 
 - `<type of work>`: This shows what kind of changes you made.
 - `<issue number>`: This is the number of the task or issue in Github Projects you're working on.
 - `<summary>`: This is a quick summary of what changes you made.
 
-Example: `doc: 159 - add login button functionality`
+❗Please note - the `summary` shouldn't contain numbers.
+
+Example:
+  your commit message `docs: add login button functionality`
+  will be automatically converted to `docs: 159 - add login button functionality`
 
 
 Please note that at RS School, we strive to follow the [Conventional Commits 1.0.0](https://www.conventionalcommits.org/en/v1.0.0/). In case of uncertainty, feel free to refer to this source that inspired us in writing these guidelines.
 
 ## Pull Requests
 
-When you finish your work in a branch, make a pull request against the main branch. Use the same format as commit messages for your pull request title.
+When you finish your work in a branch, make a pull request against the main branch. While you are editing your pull request, it will be named after your branch. After opening the pull request, it will be automatically renamed according to the template.
```

Comment:
I'm curious; shouldn't this information also somehow be duplicated in pull_request_template.md?

Reply:
It's only about PR name, this changes doesn't affected PR template. PRs' name will be changed automatically.

Reply:
I mean, the result of the automatic action of the script could be confusing - because it's rewriting whatever the user initially wrote as the title of the PR in the process of opening it. May be this should be described more?

Something like this (cahtgpt text):

> When you finish your work in a branch, make a pull request against the main branch. Initially, the pull request will be named after your branch. However, be aware that once the pull request is opened, an automatic script will rename it according to a predefined template, overwriting the title you initially provided.

Reply:
Changed